src/include/msvc.h

```diff
@@ -126,3 +126,19 @@ WT_WRITE_BARRIER(void)
 {
     _mm_sfence();
 }
+
+/*
+ * __wt_atomic_load_acquire64 --
+ *     Read a memory value with atomic acquire synchronization.
+ */
+static inline void
+__wt_atomic_load_acquire64(void *ret, void *ptr)
+{
+    /*
+     * Since we only support x86 on windows and x86 has Total Storage Ordering, no actual read
```

Comment:
"Storage" --> "Store"

Reply:
I also suggest "no actual read barrier is needed" --> "no hardware read barrier is needed". Otherwise it is confusing to say in one place we don't need a barrier then I the next sentence to say we're adding a compiler barrier.

Reply:
done